www/themes/htmx-theme/templates/base.html

```diff
@@ -52,9 +52,11 @@
                             </label>
                         </form>
                     </div>
-                </div>
-                <div class="github-stars">
-                    <iframe style="margin:auto;" src="https://ghbtns.com/github-btn.html?user=bigskysoftware&repo=htmx&type=star&count=true" frameborder="0" scrolling="0" width="150" height="20" title="Star htmx on GitHub"></iframe>
+                    <div>
+                        <div class="github-stars" hx-preserve="true" id="github-stars">
```

Comment:
Added `hx-preserve` to avoid flickering when the button is loaded for the first time